src/services/auth.js

```diff
@@ -1,14 +1,21 @@
-import axios from 'axios';
 import jwt_decode from 'jwt-decode';
+import axios from 'axios';
 import setAuthToken from './setAuthToken';
 
 const config = {
   headers: { 'Content-Type': 'application/json' },
 };
 
+export const socialLoginRequest = res => {
+  const decoded = jwt_decode(res.payload);
```

Comment:
remove jwt-decode lib from front-end. send get request for user data with this token.